plugins/github-deployments/src/components/GithubDeploymentsCard.tsx

```diff
@@ -66,10 +75,12 @@ const GithubDeploymentsComponent = ({
 };
 
 export const GithubDeploymentsCard = ({
-  last,
+  lastDeployments,
+  lastStatuses,
   columns,
 }: {
-  last?: number;
+  lastDeployments?: number;
```

Comment:
Perhaps we could keep this a non-breaking change by keeping the `last` prop name intact? It's somewhat implied that it's the last couple of deployments since it's a deployments card.